src/core_modules/capture-core/rulesEngineActionsCreator/converters/inputValueConverter.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 /* eslint-disable class-methods-use-this */
-import { moment } from 'capture-core-utils/moment';
-import type { IConvertInputRulesValue } from 'capture-core-utils/RulesEngine/rulesEngine.types';
+import { moment } from '../../../capture-core-utils/moment';
```

Comment:
can we keep those changes since my IDE throws always an error when in the path we use aliases?

Reply:
what's wrong with your IDE? 😁 What's the error? It's no biggie, we can change it.

Reply:
problem is with eslint. Dont you get the error as well?
![image](https://user-images.githubusercontent.com/4181674/80721955-20cfbc00-8aff-11ea-822b-6aada14c2563.png)